howto/setup/index.md

```diff
@@ -0,0 +1,23 @@
+# Setup
+
+See the how-to guides in this section for the steps to set up and deploy various components of Charmed HPC, such as the Slurm workload manager or a shared filesystem.
+
+- {ref}`howto-setup-deploy-slurm`
+- {ref}`howto-setup-deploy-shared-filesystem`
+
+
+
+
+
+
+
+
+:::{toctree}
+:titlesonly:
+:maxdepth: 1
+:hidden:
+
+Deploy Slurm <deploy-slurm>
+Deploy a Shared Filesystem <deploy-shared-filesystem>
```

Comment:
Lowercase since "Share Filesystem" isn't a specific thing but rather a generic concept with multiple implementations (Lustre, CephFS, NFS, etc.):

```suggestion
Deploy a shared filesystem <deploy-shared-filesystem>
```

_Grammar is fun..._